system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -326,34 +321,75 @@
 }
 
 .topsite-form {
+  $form-width: 300px;
+  $form-spacing: 32px;
+
+  .form-input-container {
+    max-width: $form-width + 3 * $form-spacing + $rich-icon-size;
+    margin: 0 auto;
+    padding: $form-spacing $form-spacing 40px;
+
+    .top-site-outer {
+      padding: 0;
+      margin: 24px 0 0;
+      margin-inline-start: $form-spacing;
+      pointer-events: none;
+    }
+
+    .section-title {
+      text-transform: none;
+      font-size: 16px;
+      margin: 0 0 16px;
+    }
+  }
+
+  .fields-and-preview {
+    display: flex;
+  }
+
+  label {
+    font-size: $section-title-font-size;
+  }
+
   .form-wrapper {
-    margin: auto;
-    max-width: 350px;
-    padding: 15px 0;
+    width: 100%;
 
     .field {
       position: relative;
+
+      .icon-clear-input {
+        position: absolute;
+        transform: translateY(-50%);
+        top: 50%;
+        offset-inline-end: 8px;
+
+        & + input:dir(ltr) {
+          padding-inline-end: 28px;
```

Comment:
Your comments and changes here made me realize this is a bit more complicate than I thought. For ltr, the padding should always be on the right, and for rtl, the padding should be on the left. This means we want padding-right and padding-left respectively because padding-inline-end will depend on the text direction of the input instead of on the document. This is particularly confusing because urls in rtl are specially treated to have text flow ltr but right aligned.